crates/league-toolkit/Cargo.toml

```
@@ -25,7 +25,8 @@ log = "0.4.21"
 memchr = "2.7.2"
 num_enum = "0.7.2"
 thiserror = "1.0.60"
-
+io-ext = { path = "../io-ext" }
+league-primitives = { path = "../league-primitives", features = ["serde"] }
```

Can we:
```suggestion
league-primitives = { path = "../league-primitives" }
```
And add that feature as a child of the crates 'serde' feature?